docker-compose.yaml

```diff
@@ -25,7 +25,7 @@ services:
   postgres:
     image: postgres:12
     ports:
-      - 5432:5432
```

Comment:
What was the motivation for this change?

Reply:
I had a running postgresql on my machine. 
Fixed.